platforms/core-runtime/stdlib-java-extensions/src/main/java/org/gradle/util/internal/TextUtil.java

```diff
@@ -338,7 +325,15 @@ public static String normaliseFileAndLineSeparators(String in) {
     }
 
     public static String camelToKebabCase(String camelCase) {
-        return KEBAB_JOINER.join(Iterables.transform(Arrays.asList(UPPER_CASE.split(camelCase)), TO_LOWERCASE));
+        StringBuilder stringBuilder = new StringBuilder();
```

Comment:
💭 I moved this class to std-java-extensions projects. It doesn't depend on Guava and Apache commons, but these dependencies are used just in these two methods

Reply:
Let's not do that. We shouldn't treat `:std-java-extensions` as the next `:base-services`. This method is not even used as-is in instrumentation; you are replacing `-` with `_` in the end. Do we even need this restructuring of the field name?

Why is `some_field_TYPE` better than `someFieldType`?

Reply:
Ah, because it's a `static` constant. Since this is generated code, I think it's perfectly fine to leave it as is. It shouldn't be checked by ErrorProne and the like if the enclosing type has the `@Generated` annotation.

Reply:
Sure, I can revert that change and put back TextUtil to base-services. I can have create a different method with a similar code in annotation processor